frontend/src/fixtures/aliasApprovalFixtures.js

```diff
@@ -0,0 +1,25 @@
+const AliasApprovalFixtures = {
+  noReview: [],
+  oneReview: [
+    {
+      id: 1,
+      proposedAlias: "CleverCat",
+    },
+  ],
+  threeReviews: [
+    {
+      id: 1,
+      proposedAlias: "CleverCat",
+    },
+    {
+      id: 2,
+      proposedAlias: "ACM",
+    },
+    {
+      id: 3,
+      proposedAlias: "SASE",
+    },
+  ],
+};
+
+export { AliasApprovalFixtures };
```

Comment:
NO!  We ALREADY have fixtures for this.  These fixtures are here:

https://github.com/ucsb-cs156-s25/proj-dining-s25-08/blob/main/frontend/src/fixtures/usersFixtures.js

Notice the fields for alias and proposedAlias?  The reason you MUST use THESE as the fixtures is that THIS is the format of the JSON that comes from the API this table MUST interact with.  You can't just make up fixtures out of thin air and expect your React component to be useful!  It has to work with a specific backend API.  The point of fixtures is to be able to design the react component even before you integrate it with the backend. But it has to be designed from the beginning to work with the backend.

Please understand this.